project.yml

```diff
@@ -30,8 +30,8 @@ targets:
     sources: [FlashSpace]
     settings:
       base:
-        MARKETING_VERSION: 2.7.34
-        CURRENT_PROJECT_VERSION: 34
+        MARKETING_VERSION: 2.7.35
+        CURRENT_PROJECT_VERSION: 35
```

Comment:
Please resolves conflicts and bump the version to `36`

Reply:
Actually, the version should be `2.8.35`. I will release it together with my changes and you PR adds a new feature so let's increase minor component.